torch/csrc/jit/pass_manager.h

```diff
@@ -19,10 +23,17 @@ namespace jit {
 // A pass modifies a Graph in place.
 using Pass = std::function<void(std::shared_ptr<Graph>&)>;
 
-TORCH_API std::vector<Pass>& getCustomPasses();
+TORCH_API std::vector<Pass>& getCustomPostFusionPasses();
+TORCH_API std::vector<Pass>& getCustomPreFusionPasses();
+
+struct TORCH_API RegisterPostFusionPass {
+  RegisterPostFusionPass(Pass p);
+};
+
+using RegisterPass = RegisterPostFusionPass;
```

Comment:
I know we have like a total of 5 (five) users of this API but I wonder if it would be better to have a compiler error than a user error as we had previously. 
To enumerate, we could:
- `RegisterPass = RegisterPostFusionPass`
- `RegisterPass = RegisterPreFusionPass` 
-  Either of the above two, and add a deprecation message to `RegisterPass`
-  Not set an alias for `RegisterPass`, so that users get a compile time error.

Any thoughts here ? It doesn't matter that much, especially if this is a temporary pass manager solution (although I wouldn't count on it being temporary).

Reply:
I don't think there is a need to compile error on this, since we can preserve the existing behavior exactly. Adding a deprecate warning is also a bit sketchy as we don't have any specific plans to actually deprecate it.

Reply:
What we did last time is silently change semantics. I think compile time errors are generally preferable.